presto-main/src/test/java/com/facebook/presto/sql/planner/TestPredicatePushdown.java

```diff
@@ -53,6 +54,15 @@
 public class TestPredicatePushdown
         extends BasePlanTest
 {
+    public TestPredicatePushdown()
+    {
+    }
```

Comment:
Its original was the implicit default empty constructor 
After adding the constructor with parameter, it will make empty constructor disappear if we don't explicitly define it